src/btreemap/node.rs

```diff
@@ -1,18 +1,24 @@
 use crate::{
-    read_struct, read_u32, read_u64,
+    btreemap::Allocator,
+    read_u32, read_u64,
     storable::Storable,
     types::{Address, Bytes},
-    write, write_struct, write_u32, Memory,
+    write, write_u32, Memory,
 };
 use std::borrow::{Borrow, Cow};
 use std::cell::{Ref, RefCell};
+use std::mem;
+
+mod v1;
+mod v2;
 
 // The minimum degree to use in the btree.
 // This constant is taken from Rust's std implementation of BTreeMap.
 const B: usize = 6;
 // The maximum number of entries per node.
 const CAPACITY: usize = 2 * B - 1;
-const LAYOUT_VERSION: u8 = 1;
+const LAYOUT_VERSION_1: u8 = 1;
+const LAYOUT_VERSION_2: u8 = 2;
```

Comment:
I guess with the new approach `UnboundedBTreeMap`, these versions will disappear. Am I right?

Reply:
Yes, correct.